### PR TITLE
Add EditorConfig file and fix related issues

### PR DESCRIPTION
**Pull request by UniQP**
Adds a basic configuration (.editorconfig) that can be handled by many editors, see https://editorconfig.org/ for more details. The PR also fixes some related whitespace issues.

### CODE REVIEW

File: main/project/attributes_okapi/AttrGen/src/GenerateAttributes.java [hunk 1]
```diff
@@ -15,133 +15,133 @@
...
-		public int oc_de_id;
-		public int oc_nl_id;
-		public int oc_pl_id;
-		public int acode;
-		public int gc_id;
-		public String[] names;
-	}
+    static class AttrInfo {
+        public int oc_de_id;
+        public int oc_nl_id;
```
[the remark this answers is not in the document]
Comment (bekuno) [reply main/project/attributes_okapi/AttrGen/src/GenerateAttributes.java:140]: You can exclude GenerateAttributes.java.
I am working currently on #7705 where I need to update it.

**Comment by Lineflyer** [comment]
Just a comment regarding the white space fixes:
IMHO we can merge them for source and translation files.
Please expect Crowdin afterwards to reset (remove) affected strings in translated files at first nonetheless as they will probably considered as changed and need new translation.

File: gradlew [hunk 2]
```diff
@@ -130,7 +130,7 @@ fi
 if [ "$cygwin" = "true" -o "$msys" = "true" ] ; then
     APP_HOME=`cygpath --path --mixed "$APP_HOME"`
     CLASSPATH=`cygpath --path --mixed "$CLASSPATH"`
-
```
Comment (bekuno) [inline gradlew:133]: gradlew and gradlew.bat are automatic created with the cradle plugin.
Please let them unchanged.

File: main/thirdparty/org/openintents/intents/FileManagerIntents.java [hunk 3]
```diff
@@ -1,130 +1,130 @@
-/* 
- * Copyright (C) 2008 OpenIntents.org
- *
```
Comment (bekuno) [inline main/thirdparty/org/openintents/intents/FileManagerIntents.java:3]: Sorry, overseen yesterday.
I think we should let this file unchanged.
The current source is at https://github.com/openintents/filemanager/blob/master/FileManager/src/main/java/org/openintents/intents/FileManagerIntents.java
Instead we should update this file with the current implementation.

**Comment by bekuno** [comment]
LGTM

**Comment by bekuno** [comment]
As info: Already discussed in #6490, but without eclipse it makes more sense.

**Comment by UniQP** [comment]
@bekuno It still makes sense if you want to edit the files with a light-weight text editor (which I regularly do).

**Comment by bekuno** [comment]
How did you analyzed the c:geo code?
I did not found an easy solution in AS while testing WhereYouGo.

**Comment by UniQP** [comment]
@bekuno I used command line tools (e.g. `git grep`) to find the files that needed adaption.

**Pull request merged**